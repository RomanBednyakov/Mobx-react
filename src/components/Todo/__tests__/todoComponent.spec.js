import * as React from 'react';
import todoComponent from '../Todo';
import renderer from 'react-test-renderer';
const onClick = jest.fn();
it('should render todoComponent', () => {
  const component = renderer.create(
    <todoComponent
      todo={{id: 1,completed: false, text:'text'}}
      onClick={onClick}
    />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
