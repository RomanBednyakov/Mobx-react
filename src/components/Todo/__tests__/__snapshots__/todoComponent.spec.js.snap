// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render todoComponent 1`] = `
<todoComponent
  onClick={[MockFunction]}
  todo={
    Object {
      "completed": false,
      "id": 1,
      "text": "text",
    }
  }
/>
`;
